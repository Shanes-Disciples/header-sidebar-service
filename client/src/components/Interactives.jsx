import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faFileAlt } from '@fortawesome/free-regular-svg-icons';
import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';

const Interactives = ({ total_downloads }) => (
  <section className="interactives-box">
    <div className="interactives">
      <div className="interactive-header">
        <strong className="interactive-header-text">Interactive Features</strong>
        <FontAwesomeIcon icon={ faInfoCircle } fixedWidth/>
      </div>
      <ul className="total-resources" >
      <li>
        <span className="feature-icon">
          <FontAwesomeIcon icon={ faFileAlt } fixedWidth/>
        </span>
        <span className="feature-text">{ `${total_downloads} downloadable resources` }</span>
      </li>
      </ul>
    </div>
  </section>
);

export default Interactives;
