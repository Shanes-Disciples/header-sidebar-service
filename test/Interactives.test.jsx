import React from 'react';
import Interactives from '../client/src/components/Interactives.jsx';
import renderer from 'react-test-renderer';

it('render correctly text component', () => {
  const InteractivesComp = renderer.create(<Interactives />).toJSON();
  expect(InteractivesComp).toMatchSnapshot();
});

