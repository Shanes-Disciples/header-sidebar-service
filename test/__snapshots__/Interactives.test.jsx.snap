// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly text component 1`] = `
<section
  className="interactives-box"
>
  <div
    className="interactives"
  >
    <div>
      <strong>
        Interactive Features
      </strong>
    </div>
    <ul
      className="total-resources"
    >
      <li>
        <span
          className="feature-icon"
        />
        <span
          className="feature-text"
        >
          undefined downloadable resources
        </span>
      </li>
    </ul>
  </div>
</section>
`;
